rest_management_api_server/operations/identity/get_identity_enrollments_urlbuilder.go

```diff
@@ -0,0 +1,123 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+//
+// Copyright NetFoundry, Inc.
```

Comment:
All these should be NetFoundry Inc. not NetFoundry, Inc. please :)

Reply:
Separate PR because it hits 1.7k files